src/operators/groupBy.ts

```diff
@@ -67,13 +77,13 @@ class GroupBySubscriber<T, R> extends Subscriber<T> {
           }
         }
 
-        this.destination.next(group);
+        this.destination.next(new GroupedObservable<R>(key, group, this.refCountSubscription));
       }
 
       if (elementSelector) {
         let value = tryCatch(elementSelector)(x);
         if (value === errorObject) {
-          group.error(value.e);
+          this.error(value.e);
```

Comment:
This single line fixes the test "Observable.prototype.groupBy() should group values with a key comparer and elementSelector, but elementSelector throws"